cns/service.go

```diff
@@ -190,6 +191,18 @@ func getTLSConfigFromFile(tlsSettings localtls.TlsSettings) (*tls.Config, error)
 		},
 	}
 
+	if tlsSettings.UseMTLS {
```

Comment:
maybe we can have a function to check if useMTLs is enabled since line 240 is doing similar tlsConfig

Reply:
i don't think this is needed as there are only two places this happens and is a niche scenario

Reply:
okay thanks for letting me know